frontend/src/widgets/HorizontalTableWidget.js

```diff
@@ -1,19 +1,20 @@
 /* eslint-disable react/no-array-index-key */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { Table } from '@trussworks/react-uswds';
+import { Table, Checkbox } from '@trussworks/react-uswds';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import colors from '../colors';
 import './HorizontalTableWidget.scss';
 
-/*
 const trimLongURLs = (url) => {
   let newUrl = url;
-  if (newUrl.length >= 35) {
-    newUrl = newUrl.substring(0, 35);
+  if (newUrl.length >= 55) {
+    newUrl = newUrl.substring(0, 55);
```

Comment:
we could do this with CSS, might save us some electricity. Something like

```css
a {
 white-space: nowrap;
  width: 55ch; /* the ch unit is character, so a max width of 55 characters */
  overflow: hidden;
  text-overflow: ellipsis;
  width:
}

Reply:
Keeping the lights on a bit longer.